library/board.py

```diff
@@ -0,0 +1,131 @@
+import typing
+
+import blessed
+import numpy as np
+import numpy.typing as npt
+
+
+class Board:
+    """Represents the board that the game of tic-tac-toe is played on."""
+
+    contents: npt.NDArray[np.str_]
+    player1_turn: bool
+    game_over: bool
+    turn_one: bool
+
+    def __init__(self) -> None:
+        self.contents = np.full((9, 9), "·")
+        self.player1_turn = True
+        self.game_over = False
+        self.turn_one = True
+
+    def collect_subgrid(self, number: str) -> npt.NDArray[np.str_]:
+        """Takes a subgrid choice and returns that np.ndarray and the choice str"""
+        subgrid = np.full((3, 3), "·")
+        if number == "0":
+            subgrid = self.contents[6:12, :3]
+        elif number == "1":
+            subgrid = self.contents[6:12, 3:6]
+        elif number == "2":
+            subgrid = self.contents[6:12, 6:12]
+        elif number == "3":
+            subgrid = self.contents[:3, :3]
+        elif number == "4":
+            subgrid = self.contents[:3, 3:6]
+        elif number == "5":
+            subgrid = self.contents[:3, 6:12]
+        elif number == "6":
+            subgrid = self.contents[3:6, :3]
+        elif number == "7":
+            subgrid = self.contents[3:6, 3:6]
+        elif number == "8":
```

Comment:
Could the numbering be documented in the docstring? Something like this:
```
The subgrids are laid out as so:

  0 1 2
  3 4 5
  6 7 8
```

Reply:
Added a line in the docstring to make that more clear:
```
    def collect_subgrid(self, number: str) -> npt.NDArray[np.str_]:
        """
        Takes a subgrid choice and returns that np.ndarray and the choice str

        The subgrids are laid out as so:
            6 7 8
            3 4 5
            0 1 2
        """
```